templates/repo-management.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+# vim: ft=sh
+set -e
+
+function cloneOrFetchNimbusRepo() {
+  echo ">>> starting cloneOrFetchNimbusRepo() inside repo-management.sh"
```

Comment:
All of these echo's are kinda pointless, but if you really want them why not just use a wrapper?
```bash
function call_func {
    echo ">>> starting ${1}"
    ${1}
    echo "<<< ending ${1}"
}

function some_work {
    echo "WORKING"
}

call_func 'some_work'
```
```
 > ./test.sh
>>> starting some_work
WORKING
<<< ending some_work
```